doc/devops/docker/README.md

```diff
@@ -0,0 +1 @@
+![docker setup](drawio_docker.png "Logo Title Text 1")
```

Comment:
We (BigchainDB GmbH) are running a Secret Store network in AWS to go with the Ocean Testnet (and similarly for the Kovan Testnet), and I'm not sure how those fit into this picture. Maybe they are just other options in the lower-left bubble? One must be careful to ensure that the chosen Secret Store is hooked up to the chosen Ethereum network.